supportedBackends/google/pipelines/v2beta/src/main/scala/cromwell/backend/google/pipelines/v2beta/api/request/GetRequestHandler.scala

```diff
@@ -135,8 +135,14 @@ trait GetRequestHandler { this: RequestHandler =>
     }
 
     // Map action indexes to event types. Action indexes are 1-based for some reason.
+    // BA-6455: since v2beta version of Life Sciences API, `a.getLabels` would return `null` for empty labels, unlike
+    // v2alpha1 version, where it returned empty list in the same case
     val actionIndexToEventType: Map[Int, String] = List(Key.Logging, Key.Tag).flatMap { k =>
-      actions.zipWithIndex collect { case (a, i) if a.getLabels.containsKey(k) => (i + 1) -> a.getLabels.get(k) } } toMap
+      actions.zipWithIndex collect {
+        case (a, i) if Option(a.getLabels).getOrElse(Map.empty[String, String].asJava).containsKey(k) =>
+          (i + 1) -> a.getLabels.get(k)
+      }
```

Comment:
Very small comment. Might be slightly more readable as:
```
      actions.map(a => Option(a.getLabels)).zipWithIndex collect {
        case (Some(labels), i) if labels.containsKey(k) => (i + 1) -> labels.get(k)
```

What do you think?

Reply:
Looks good, thanks.